client/my-sites/email/inbox/mailbox-selection-list/progress-line-style.scss

```diff
@@ -0,0 +1,101 @@
+@import '@wordpress/base-styles/breakpoints';
+@import '@wordpress/base-styles/mixins';
+
+.progress-line__container {
+	position: fixed;
+	top: 50%;
+	left: 50%;
+	transform: translate( -50%, -50% );
+
+	@include break-large {
+		transform: none;
+	}
+
+
+	.progress-line__status-text-container {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		margin-bottom: 10px;
+
+		@include break-large {
+			margin-bottom: 20px;
+		}
+
+		.progress-line__status-text {
+			font-family: $sans;
+			font-weight: 400;
+			font-size: $font-title-small;
+			line-height: 22px;
+
+			@include break-large {
+				font-size: $font-title-medium;
+				line-height: 28px;
+			}
+		}
+	}
+
+	.progress-line__loading-container {
+		position: relative;
+		width: 280px;
+		margin: 0 auto;
+		border-radius: 2px;
+		box-sizing: border-box;
+
+		@include break-medium {
+			width: 335px;
+		}
+
+		@include break-large {
+			width: 400px;
+		}
+
+		@keyframes load {
+			0% {
+				width: 0;
+				background: var( --studio-gray-70 );
+			}
+
+			25% {
+				width: 40%;
+				background: var( --studio-gray-70 );
+			}
+
+			50% {
+				width: 60%;
+				background: var( --studio-gray-70 );
+			}
```

Comment:
Is there a particular reason these are 40% and 60% instead of 25% and 50%? This leads to weird/uneven jumps.